parse.lisp

```diff
@@ -227,7 +234,8 @@
   (check-type arg simple-string)
   (and (<= 2 (length arg))
        (or (char= #\- (schar arg 0))
-           (char= #\+ (schar arg 0)))))
+           (char= #\+ (schar arg 0)))
+       (not (find #\, arg))))
```

Comment:
What is special about the comma character ?

Reply:
Nothing. Will revert.